README.md

```diff
@@ -163,7 +163,7 @@ Depending on the platform, this will create:
 
 The artifact is written to the `release` directory.
 
-## Publishing release artifacts
+### Publishing release artifacts
```

Comment:
Maybe we should remove this section.

Reply:
I removed both sections `Creating release artifacts` and `Publishing release artifacts` in 6954316702140d76aba00926fb085dda0404f69e. I thought about moving them into the Nordic wiki, since they are only done by Nordic employees, but we already have https://projecttools.nordicsemi.no/confluence/display/WYLND/nRF+Connect+release+procedures and this seemed sufficient to me.